src/pages/s/[classID]/teachers.tsx

```diff
@@ -0,0 +1,362 @@
+                value: "",
+              },
+            ],
+          },
+        },
+      ],
+    },
+    {
+      groupName: "วิทยาศาสตร์",
+      content: [
+        {
+          id: 3,
+          content: {
+            id: 3,
+            prefix: "master",
+            name: {
+              "en-US": { firstName: "Thanakorn", lastName: "Arjanawut" },
+              th: { firstName: "ธนกร", lastName: "อรรจนาวัฒน์" },
+            },
+            profile: "/images/dummybase/thanakorn.webp",
+            role: "teacher",
+            classAdvisorAt: {
+              id: 1,
+              name: {
+                "en-US": "M.405",
+                th: "ม.405",
+              },
+            },
+            subjectsInCharge: [
+              {
+                id: 3,
+                code: {
+                  "en-US": "SCI6969",
+                  th: "ว6969",
+                },
+                name: {
+                  "en-US": {
+                    name: "Core Science",
+                  },
+                  th: { name: "วิทยาศาสตร์พื้นๆ" },
+                },
+                subjectSubgroup: {
+                  name: { "en-US": "Science", th: "วิทยาศาสตร์" },
+                  subjectGroup: {
+                    name: { "en-US": "Science", th: "วิทยาศาสตร์" },
+                  },
+                },
+              },
+              {
+                id: 4,
+                code: {
+                  "en-US": "SCI42069",
+                  th: "ว42069",
+                },
+                name: {
+                  "en-US": {
+                    name: "Additional Science",
+                  },
+                  th: { name: "วิทยาศาสตร์เพิ่มเติม" },
+                },
+                subjectSubgroup: {
+                  name: { "en-US": "Science", th: "วิทยาศาสตร์" },
+                  subjectGroup: {
+                    name: { "en-US": "Science", th: "วิทยาศาสตร์" },
+                  },
+                },
+              },
+            ],
+            contacts: [
+              {
+                id: 3,
+                name: {
+                  "en-US": "xx_ธนกร_อรรจนาวัฒน์_xx",
+                  th: "xx_ธนกร_อรรจนาวัฒน์_xx",
+                },
+                via: "facebook",
+                value: "",
+              },
+              {
+                id: 4,
+                name: {
+                  "en-US": "ธนกร_อรรจนาวัฒน์xd",
+                  th: "ธนกร_อรรจนาวัฒน์xd",
+                },
+                via: "line",
+                value: "",
+              },
+              {
+                id: 5,
+                name: {
+                  "en-US": "Thanacord",
+                  th: "Thanacord",
+                },
+                via: "discord",
+                value: "",
+              },
+            ],
+          },
+        },
+      ],
+    },
+  ];
+
+  const [showMain, setShowMain] = useState(teacherList[0].content[0].content);
+
+  showMain.contacts.sort(function (a, b) {
```

Comment:
normally we would use arrow function for anonymous function but it will be very similar

Reply:
and this function can be shorten with ternary operator
```ts
showMain.contacts.sort((a, b) => a.via < b.via ? -1 : a.via > b.via ? 1 : 0)
```